model_service.py

```diff
@@ -0,0 +1,101 @@
+from abc import ABCMeta, abstractmethod
+
+
+class ModelService(object):
+    """ModelService wraps up all preprocessing, inference and postprocessing
+    functions used by model service. It is defined in a flexible manner to
+    be easily extended to support different frameworks.
+    """
+    __metaclass__ = ABCMeta
+
+    def __init__(self, path, ctx):
+        self.context = ctx
+
+    @abstractmethod
+    def predict(self, data):
+        pass
+
+
+class SingleNodeModel(ModelService):
```

Comment:
How about rename to SingleNodeModelService？Or SingleNodeService？

Reply:
Yes. Will change it.